src/modelmanager.cpp

```diff
@@ -579,19 +579,18 @@ Status ModelManager::loadCustomLoadersConfig(rapidjson::Document& configJson) {
 }
 
 Status ModelManager::loadMetricsConfig(rapidjson::Document& configJson) {
-    Status firstErrorStatus = StatusCode::OK;
+    Status status = StatusCode::OK;
     const auto itr2 = configJson.FindMember("monitoring");
     if (itr2 == configJson.MemberEnd() || !itr2->value.IsObject()) {
         SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Configuration file doesn't have monitoring property.");
     } else {
         const auto& metrics = itr2->value.GetObject();
         SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Parsing monitoring metrics config settings.");
         bool forceFailureIfMetricsAreEnabled = ovms::Config::instance().restPort() == 0;
-        firstErrorStatus = this->metricConfig.parseMetricsConfig(metrics, forceFailureIfMetricsAreEnabled);
-        IF_ERROR_NOT_OCCURRED_EARLIER_THEN_SET_FIRST_ERROR(firstErrorStatus);
+        status = this->metricConfig.parseMetricsConfig(metrics, forceFailureIfMetricsAreEnabled);
```

Comment:
Why keep variable in line 582 if you can return here right away?

Reply:
You need status if you dont go into else